LogProcessing.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Mar 15 17:07:04 2019
+
+@author: Nataliya_Pavych
+"""
+
+
+def load_file_into_list_by_line(file_name: str) -> object:
+    with open(file_name) as text_file:
+        return [line for line in text_file]
+
+
+def clean_data_into_d2_list(list_name):
+    return list(map(lambda string: (lambda x, y, z: [f'{x} {y[1:12]}', int(z)])(*string.split(' ')), list_name))
```

Comment:
Please consider rewriting this with a plain for loop, this greatly improves readability.